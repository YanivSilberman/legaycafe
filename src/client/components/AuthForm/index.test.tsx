import * as React from 'react';
import * as ReactShallowRenderer from 'react-test-renderer/shallow'
import Component from './index';

describe('AuthForm component test',() => {
  it('component loads', () => {
    const renderer = ReactShallowRenderer.createRenderer();

    expect(renderer.render(
      <Component />
    )).toMatchSnapshot()
  })
});
