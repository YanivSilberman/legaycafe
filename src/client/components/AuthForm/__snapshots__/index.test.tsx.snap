// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthForm component test component loads 1`] = `
<withRouter(withApollo(Apollo(AuthForm)))
  classes={
    Object {
      "authForm": "withRouter(withApollo(Apollo(AuthForm)))-authForm-1",
      "button": "withRouter(withApollo(Apollo(AuthForm)))-button-3",
      "textField": "withRouter(withApollo(Apollo(AuthForm)))-textField-2",
    }
  }
  client={
    Object {
      "resetStore": [Function],
    }
  }
  history={Object {}}
  loginUserMutation={[Function]}
/>
`;
